chia/wallet/wallet_nft_store.py

```diff
@@ -87,22 +124,29 @@ async def save_nft(self, wallet_id: uint32, did_id: Optional[bytes32], nft_coin_
                     int(nft_coin_info.latest_height),
                 ),
             )
-            await cursor.close()
             # Rotate the old removed NFTs, they are not possible to be reorged
-            await (
-                await conn.execute(
-                    "DELETE FROM users_nfts WHERE removed_height is not NULL and removed_height<?",
-                    (int(nft_coin_info.latest_height) - REMOVE_BUFF_BLOCKS,),
-                )
-            ).close()
+            await conn.execute(
+                "DELETE FROM users_nfts WHERE removed_height is not NULL and removed_height<?",
+                (int(nft_coin_info.latest_height) - REMOVE_BUFF_BLOCKS,),
+            )
+
+    async def count(self, wallet_id: Optional[uint32] = None, did_id: Optional[bytes32] = None) -> int:
+        sql = "SELECT COUNT(nft_id) FROM users_nfts WHERE removed_height is NULL"
+        if wallet_id is not None:
```

Comment:
the test of `is not None` is inconsistent here and on line 137 (which uses the shorter form). I think they should be consistent, and I'm not sure whether we have a policy, or whether pylint/flake8 has opinons